src/main/java/de/uni_tuebingen/ub/nppm/model/SelektionTitelKritik.java

```diff
@@ -0,0 +1,31 @@
+package de.uni_tuebingen.ub.nppm.model;
+
+import javax.persistence.*;
+import java.util.*;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+@Entity
+@Table(name = "selektion_titelkritik")
+@Cacheable
+@org.hibernate.annotations.Cache(usage = CacheConcurrencyStrategy.READ_WRITE)
+public class SelektionTitelKritik extends SelektionBezeichnung{
+
+     @ManyToMany(mappedBy = "titelKritiken")
+    private Set<Einzelbeleg> einzelbeleg = new HashSet<>();
+
+    public Set<Einzelbeleg> getEinzelbeleg() {
+        return einzelbeleg;
+    }
+
+    public void setEinzelbeleg(Set<Einzelbeleg> einzelbeleg) {
+        this.einzelbeleg = einzelbeleg;
+    }
+
+     public void addEinzelbeleg(Einzelbeleg beleg) {
```

Comment:
Please watch your indentations, seems to be wrong in multiple occasions throughout the code.

Reply:
done